src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,311 @@
+# Common data types
+
+## Processed, coregistered and/or resampled volumes
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        func|anat|dwi/
+        <source_keywords>[_space-<space>][_desc-<label>]_<suffix>.<ext>
+```
+
+Processing in this context means transformations of data that does not change
+the number of dimensions of the input and are not explicitly covered by other
+data types in the specification. Examples:
+
+-   Motion-corrected, temporally denoised, and transformed to MNI space bold
+    files.
+
+-   Inhomogeneity corrected and skull stripped T1w files.
+
+-   Motion-corrected DWI files.
+
+The `space` keyword is recomended to distinguish files with different underlying
+coordinate systems or registered to different reference maps. The
+`desc` keyword is a general purpose field with freeform values. To distinguish
+between multiple different versions of processing for the same input data the
+`desc` keyword should be used. Note that even though `space` and `desc` are
```

Comment:
perhaps the note can be a new paragraph to give attention to this point

Reply:
```suggestion
`desc` keyword should be used.

Note that even though `space` and `desc` are
```